libs/host/Configuration/Options.cs

```diff
@@ -64,11 +64,11 @@ internal sealed class Options
         public int MutablePercent { get; set; }
 
         [MemorySizeValidation(false)]
-        [Option("obj-total-memory", Required = false, HelpText = "Total object store log memory used including heap memory in bytes")]
-        public string ObjectStoreTotalMemorySize { get; set; }
+        [Option("obj-heap-memory", Required = false, HelpText = "Object store heap memory size in bytes")]
```

Comment:
explain what heap memory means - "Object store heap memory size (to account for the size taken up by object instances on the heap)"